ckanext/ontario_theme/templates/package/snippets/ontario_theme_language_disclaimer.html

```diff
@@ -0,0 +1,23 @@
+{#
+  Custom language notification in the cases that users are viewing the site in one language and a resource in another
+#}
+{% set current_lang = request.environ.CKAN_LANG %}
+{% if (res.language == "english" and current_lang == "fr") or (res.language == "french" and current_lang == "en") %}
```

Comment:
For future time, I think it would be good to walkthrough these conditionals. Essentially wondering if they are needed or add an extra layer of complexity for us and the user.  I had imagined these would just always show (e.g. "this resource is in [language x] only.").